src/main/resources/application.properties

```diff
@@ -86,6 +86,8 @@ nae.server.security.realm-name=NAE-Realm
 nae.server.security.headers.hsts.enable=true
 nae.server.security.headers.hsts.include-sub-domains=true
 nae.security.limits.login-attempts=50
+nae.security.limits.login-timeout=1
+nae.security.limits.login-timeout-unit=DAYS
```

Comment:
the properties values do not match the defaults in the `SecurityLimitsProperties` class, but match the previous time-out duration from `LoginAttemptService`

Is this intentional?

Reply:
time from https://github.com/netgrif/application-engine/pull/173/files/288ba1d05a14b90e996393731aa154cc059bcc4a#diff-24d326f12fc91b994d87770e48328482375b8740e4fe51c58e21c939177962a3L28